test/unit/comment_test.rb

```diff
@@ -16,7 +16,12 @@ class CommentTest < ActiveSupport::TestCase
       assert comments.length > 0
     end
   end
-
+  test 'should have gmail quote' do
+    mail = Mail.read('test/fixtures/incoming_test_emails/gmail/incoming_gmail_email.eml') 
+    mail_doc = Nokogiri::HTML(mail.html_part.body.decoded) # To parse the mail to extract comment content and reply content 
+    gmail_quote = Comment.gmail_quote_present?(".gmail_quote")
```

Comment:
@dinaelhanan You forgot to change the argument :joy:

Reply:
ooh woops!